CIPs/0022.md

```diff
@@ -0,0 +1,40 @@
+---
+cip: 0022
+title: Standard for Injacted web3 Provider
+author: @daoauth @NAKsir-melody
+discussions-to: https://github.com/celo-org/celo-proposals/issues/74
+status: Draft
+type:
+category:
```

Comment:
@prestwich seems to me this is a Ring 4 category due to specifying RPC endpoints that don't really impact the network per se. Any thoughts on it?

Reply:
Ring 3, I believe

Reply:
@daoauth can you update type to `Informational` and category to `Ring 3`? I will merge as a draft after that